README.md

```diff
@@ -0,0 +1,13 @@
+# testproj - a test project for testing github actions, also used as Respository Under Test (RUT) for gpt-commit-summarizer
```

Comment:
GPT summary of [None](https://github.com/d4nshields/testproj/blob/968c75e5cb31c940ccc17310a601c9d8331a294a/README.md#None) - [0c10cd](https://github.com/d4nshields/testproj/blob/a903389db4c6cded0ad4668f9e87760b2339c41f/README.md#0c10cd228621bc095cf560d865f63c301c7f7774):
* Added a description of the project, the author, and the license.